src/Microsoft.AspNetCore.Mvc.ViewFeatures/ExpressionTextCache.cs

```diff
@@ -1,124 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Linq.Expressions;
-using Microsoft.Extensions.Internal;
-
-namespace Microsoft.AspNetCore.Mvc.ViewFeatures
-{
-    /// <summary>
-    /// This class holds the cache for the expression text that is computed by ExpressionHelper.
-    /// </summary>
-    public class ExpressionTextCache
```

Comment:
Extracted `LambdaExpressionComparer` into a separate `internal` type as-is. Git doesn't like the diff so it's treating this as a add-delete